runtime/syntax/gdscript.yaml

```
@@ -12,15 +12,18 @@ rules:
     # Built-in node names
     - identifier: "\\b(AnimationPlayer|AnimationTreePlayer|Button|Control|Engine|HTTPClient|HTTPRequest|Input|InputEvent|MainLoop|Node|Node2D|OS|SceneTree|Spatial|StreamPeer|PacketPeer|PacketPeerUDP|Timer|Tween)\\b"
     # Types
-    - type: "\\b(AABB|Array|Basis|Color|Dictionary|NodePath|Object|Plane|PoolByteArray|PoolColorArray|PoolIntArray|PoolRealArray|PoolVector2Array|PoolVector3Array|Quat|Rect2|RID|String|Transform|Transform2D|Vector2|Vector3)\\b"
+    - type: "\\b(AABB|Array|Basis|Color|Dictionary|NodePath|Object|Plane|PoolByteArray|PoolColorArray|PoolIntArray|PoolRealArray|PoolVector2Array|PoolVector3Array|Quat|Rect2|RID|String|Transform|Transform2D|Vector2|Vector3|void)\\b"
     # Definitions
     - identifier: "func [a-zA-Z_0-9]+"
     # Keywords
-    - statement: "\\b(and|as|assert|break|breakpoint|class|const|continue|elif|else|enum|export|extends|for|func|if|in|is|map|master|mastersync|match|not|onready|or|pass|remote|remotesync|return|self|setget|slave|slavesync|signal|sync|tool|var|while|yield)\\b"
+    - statement: "\\b(and|as|assert|await|break|breakpoint|class|class_name|const|continue|elif|else|enum|export|extends|for|func|if|in|is|map|master|mastersync|match|not|onready|or|pass|remote|remotesync|return|self|setget|slave|slavesync|signal|sync|tool|var|while|yield)\\b"
 
     # Operators
     - statement: "[.:;,+*|=!\\%@]|<|>|/|-|&"
 
+    # Annotations
+    - special: "^@\\b(export|export_category|export_color_no_alpha|export_dir|export_enum|export_exp_easing|export_file|export_flags|export_flags_2d_navigation|export_flags_2d_physics|export_flags_2d_render|export_flags_3d_navigation|export_flags_3d_physics|export_flags_3d_render|export_flags_avoidance|export_global_dir|export_global_file|export_group|export_multiline|export_node_path|export_placeholder|export_range|export_subgroup|icon|onready|rpc|tool|warning_ignore)\\b+"
```

Comparing with [Godot docs](https://docs.godotengine.org/en/stable/classes/class_%40gdscript.html#annotations) the `@static_unload` annotation seems to be missing. The first `\\b` and the `+` at the end don't do anything useful.

```suggestion
    - special: "^@(export|export_category|export_color_no_alpha|export_dir|export_enum|export_exp_easing|export_file|export_flags|export_flags_2d_navigation|export_flags_2d_physics|export_flags_2d_render|export_flags_3d_navigation|export_flags_3d_physics|export_flags_3d_render|export_flags_avoidance|export_global_dir|export_global_file|export_group|export_multiline|export_node_path|export_placeholder|export_range|export_subgroup|icon|onready|rpc|static_unload|tool|warning_ignore)\\b"
```

---

1. `+` - typo
2. `@static_unload` - thanks
3. `\\b` It is used everywhere. Something starts from `\\b` and ends from it

---

> \\b It is used everywhere. Something starts from \\b and ends from it

`\b` means word boundary (for example `\bas\b` would match both `"as"` and `"@as"` but not `"gas"`). In this case it is not necessary because there is *always* a word boundary (the previous character is a non-word character `@` and the following character is a letter).